functions-and-scope.js

```diff
@@ -52,26 +82,57 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade(grades) geeft 6.642857142857143
 // averageGrade([6, 4, 5]) geeft 5
 // averageGrade([8, 9, 4, 6, 10]) geeft 7.4
+function averageGrade(grades) {
+    let averageGradeNumber = 0;
+
+    for (let i = 0; i < grades.length; i++) {
+        averageGradeNumber += grades[i];
+    }
+    let average = averageGradeNumber / grades.length;
```

Comment:
Je zou ook direct het gemiddelde kunnen retourneren in plaats van eerst een variabele te maken en die vervolgens te retourneren.

Reply:
Bedankt voor je feedback :)